src/main/java/cd/go/contrib/elasticagent/KubernetesInstanceFactory.java

```diff
@@ -74,6 +75,14 @@ private KubernetesInstance create(CreateAgentRequest request, PluginSettings set
         return createKubernetesPod(client, elasticAgentPod);
     }
 
+    private Boolean privileged(CreateAgentRequest request) {
+        final String privilegedMode = request.properties().get(PRIVILEGED.getKey());
+        if (StringUtils.isBlank(privilegedMode)) {
+            return false;
```

Comment:
If blank, should we default to true or false?

Reply:
It should be false. You set it to true to run in privileged mode. Otherwise if you have old profile without this property will not start container in privileged mode.